fhir_zulip_nlp/fhir_zulip_nlp.py

```diff
@@ -27,7 +27,7 @@
     'zuliprc_path': os.path.join(PROJECT_DIR, 'zuliprc'),  # rc = "runtime config"
     'outpath': os.path.join(PROJECT_DIR, 'zulip_report.csv')
 }
-
+CLIENT = zulip.Client(config_file=CONFIG['zuliprc_path'])
```

Comment:
Moved this to the top of the file because it only needs to be declared once.

There is a common convention in Python which I use for constants. Any variables that I declare at the root scope of a script (outside of any functions), I use all caps. This indicates that the variable is intended to be used as a constant, and it should not be altered at runtime.